src/ui/NoNameFacet/NoNameFacetValues/NoNameFacetValueCheckbox.ts

```diff
@@ -0,0 +1,27 @@
+import { $$ } from '../../../utils/Dom';
+import { SVGIcons } from '../../../utils/SVGIcons';
+import { SVGDom } from '../../../utils/SVGDom';
+
+export class NoNameFacetValueCheckbox {
+  public element: HTMLElement;
+
+  constructor(private label: string) {
+    this.create();
+  }
+
+  private create() {
```

Comment:
It should be using a semantic `input type=checkbox` element.

I think simply using `src/ui/FormWidgets/Checkbox` would be 👌